frontend/dashboard/components/SafeErrorView/SafeErrorView.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { SafeErrorView } from './SafeErrorView';
+import { textMock } from '../../../testing/mocks/i18nMock';
+
+const { reload: originalReload } = window.location;
+
+describe('SafeErrorView', () => {
+  beforeEach(() => {
+    Object.defineProperty(window, 'location', {
+      writable: true,
+      value: { reload: jest.fn() },
+    });
+  });
+
+  afterEach(() => {
+    window.location.reload = originalReload;
+  });
+
+  it('should render heading, title and message, with a relaod button', () => {
+    render(
+      <SafeErrorView
+        heading='All users list'
```

Comment:
Can this be refactored into a variable? I can see that it used several places. The same goes for title and message 😄

Reply:
I have made it a little more DRY, but I'm quite a huge fan of keeping the data explicit for each test. In my opinion, it increases the readability in the context of testing. 

Look at the code now after I committed the latest changes. You will see that the first test is using the texts that are sent in. But the second test does not care about any texts, just the reload button.

Reply:
I believe there might be some misunderstanding between us 😅 I added the comment on the place I ment 😄 
I did not mean to move the component out, but it looks really nice and clean how it is done now! 🚀